pilot/lib/pilot/tester_manager.rb

```diff
@@ -91,21 +95,41 @@ def find_app(app_filter: nil)
       nil
     end
 
-    def find_or_create_tester(email: nil, first_name: nil, last_name: nil)
-      tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
-      tester ||= Spaceship::Tunes::Tester::External.find(config[:email])
+    def find_or_create_tester(email: nil, first_name: nil, last_name: nil, app: nil)
```

Comment:
Totes agree. I started to decompose it early and then stuck it all back together when it stopped working. I think it's totally a good thing to do though.